imgui_widgets.cpp

```diff
@@ -720,6 +720,61 @@ bool ImGui::Button(const char* label, const ImVec2& size_arg)
     return ButtonEx(label, size_arg, ImGuiButtonFlags_None);
 }
 
+bool ImGui::BeginButtonDropDown(const char* label, ImVec2 buttonSize)
+{
+    ImGui::SameLine(0.f, 0.f);
```

Comment:
I don't think this should be here, calling `SameLine` is the user's  responsibility.  What happens if you don't want it to be on the same line?

Reply:
This doesn't make sense given how drop-downs work. The drop-down isn't really appropriate on the next line. It is inherently a horizontal feature.

Reply:
So I can't put a dropdown button below something if I want to?

Reply:
![image](https://user-images.githubusercontent.com/81622310/171996813-2863fa6e-8472-44aa-97de-77803155ffe2.png)
We can't do ths.